Code/Framework/AzFramework/Platform/Common/Xcb/azframework_xcb_files.cmake

```diff
@@ -6,6 +6,10 @@
 #
 #
 
+# generate logo.h, based on logo.png
+execute_process(COMMAND xxd -i logo.png logo.h
+                WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/AzFramework)
```

Comment:
This still does not alleviate the concern of having a project-specific splash screen. The logo.png is still in the engine source. This approach also will write the generated `logo.h` file back to `AzFramework` path, and that by itself raises a some issues:
* The folder is under git control, so the file will be marked as a new file
* For installer builds, it generates a file into the installation files, which generally is not preferred
* For Linux SNAP packages, this will fail because the engine folder is in an immutable folder structure, and therefore cannot be written anyways.

Reply:
Also `xxd` needs to be in prerequisites for linux build.

Reply:
Thank you for pointing those issues, I think now I resolved them, I'm using cmake to generare this splash file, from Splash.png located in Project resources folder, to build directory, so no git controlled folder is touched, also it should solve with SNAP. 
This approach also allows for per project logo specification as it takes data from project files